apps/store/extensions/app/store-common/pages/top_assets.jag

```diff
@@ -58,6 +58,12 @@ require('/modules/store.js').exec(function (ctx) {
     }*/
     page.assetMeta.q = q;
     var output = appManager.render([], page);
+    var recentAssets = output.recentAssets;
+    for(var i in recentAssets){
+        if(recentAssets[i].thumbnail == 'null'){
```

Comment:
Done in : https://github.com/chamindias/carbon-store/commit/824e2002bcf658058345ba2d3e3b834751366f1c